engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -125,6 +126,8 @@
 	 */
 	private final List<ValidatedValueUnwrapper<?>> validatedValueHandlers;
 
+	private final ClassLoader userClassLoader;
```

Comment:
> isn't everything provided what is passed to the config

Right, so it is making clear that this is not a class loader which we have access to per default, but it is "provided" to us during configuration.

> Maybe externalClassLoader

Could work. Better than userClassLoader imo

> userResourceClassLoader

-1

I think providedClassLoader is best :-)